LibDeflate.lua

```diff
@@ -2732,13 +2740,14 @@ end
 -- If the decompression fails (The first return value of this function is nil),
 -- this return value is undefined.
 -- @see LibDeflate:CompressDeflate
-function LibDeflate:DecompressDeflate(str)
-	local arg_valid, arg_err = IsValidArguments(str)
+function LibDeflate:DecompressDeflate(str, configs)
```

Comment:
Add some tests in tests/Test.lua, which should covers all new code in LibDeflate.lua